testing/workflows/components/workflows.libsonnet

```diff
@@ -278,6 +278,25 @@
         },
         dependencies: ["wait-for-kubeflow"],
       },  // pytorchjob - deploy,
+      {
```

Comment:
I think this is legacy; I think this workflow is only used by the minikube test because it hasn't been updated to use kfctl.sh. 

I think its fine not to run the test on minikube.

Reply:
So this part seems to be triggered outside of the minikube tests.

That being said, I think it might make sense, in the future, to test this and other operator one-by-one on minikube, what do you think?